common/src/main/java/org/dash/wallet/common/data/ExchangeRatesConfig.kt

```diff
@@ -0,0 +1,25 @@
+package org.dash.wallet.common.data
+
+import android.content.Context
+import androidx.datastore.preferences.core.booleanPreferencesKey
+import androidx.datastore.preferences.core.longPreferencesKey
+import org.dash.wallet.common.WalletDataProvider
+import javax.inject.Inject
+
+class ExchangeRatesConfig
+    @Inject
+    constructor(
+        context: Context,
+        walletDataProvider: WalletDataProvider
+    ) : BaseConfig(
+            context,
+            PREFERENCES_NAME,
+            walletDataProvider
+        ) {
+        companion object {
+            private const val PREFERENCES_NAME = "exchange_rates_config"
+            val EXCHANGE_RATES_RETRIEVAL_TIME = longPreferencesKey("exchange_rates_retrieval_time")
+            val EXCHANGE_RATES_RETRIEVAL_FAILURE = booleanPreferencesKey("exchange_rates_retrieval_error")
+            val EXCHANGE_RATES_PREVIOUS_RETRIEVAL_TIME = longPreferencesKey("exchange_rates_previous_retrieval_time")
+        }
+    }
```

Comment:
At first I was going to put these in WalletUIConfig, but that would have created a circular reference.

Reply:
`WalletUIConfig` might be better to leave for UI settings that can be changed by the user, or closely related preferences.